lab/app/views/samples/edit.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= form_with(model: @sample, local: true) do |form| %>
+       
+    <table>
+    <tr>
+        <td>
+        <%= form.label :sample_type %><br>
+        <%= form.text_field :sample_type %>
+        </td>
+        
+    </tr>
+
+    <tr>
+        <td>
+        <%= form.label :collect_date %><br>
+        <%= form.text_field :collect_date %>
```

Comment:
esse campo fica melhor com um [select de data](https://guides.rubyonrails.org/form_helpers.html#using-date-and-time-form-helpers-model-object-helpers)